src/components/PeoplePage.tsx

```diff
@@ -1,34 +1,198 @@
-import { PeopleFilters } from './PeopleFilters';
+import {
+  useCallback, useEffect, useMemo, useState,
+} from 'react';
+import { useParams, useSearchParams } from 'react-router-dom';
+import { Person } from '../types';
+import { getPeople } from '../api';
+import { PeaopleTable } from './PeopleTable';
```

Comment:
There is a typo in the component name. It should be 'PeopleTable' instead of 'PeaopleTable'.